adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -405,7 +418,7 @@ - (void)completeBannerAdViewForPlacementID:(nonnull id<GADMAdapterVungleDelegate
       return;
     }
 
-    if (_isBannerPresenting || (delegate.bannerState == BannerRouterDelegateStatePlaying)) {
+    if ((_isBannerPresenting && delegate.bannerState == BannerRouterDelegateStatePlaying) || delegate.bannerState == BannerRouterDelegateStateWillPlaying) {
```

Comment:
Only the Banner delegate state is in `BannerRouterDelegateStatePlaying` or  `BannerRouterDelegateStateWillPlaying` (which represents the state that the Banner view is successfully returned by calling `addAdViewToView:withOptions:placementID:placementID:error:` method) , the adapter will call `[[VungleSDK sharedSDK] finishedDisplayingAd]` method.  Thanks. ^_^

Reply:
One question here - if the states are `...WillPlay` (meaning the adapter has the ad view returned by the SDK) and `...Playing` (meaning the ad is visible on the screen), why would we allow the adapter to close the ad experience if it has never been presented? Is there a use case where the ad will be displayed but the state will not be properly updated?

Reply:
@crbinsf Hi Clarke, in a scenario that the Banner view is successfully returned by calling `addAdViewToView:withOptions:placementID:placementID:error:`  method, but before displaying the Banner ad, the Banner View's super view is removed, such as the user  exits the view controller which plays the Banner Ad. Thanks.  ：）